.pre-commit-config.yaml

```diff
@@ -79,7 +79,7 @@ repos:
       - id: yamllint
         args: ["--strict", "--config-file=.yamllint.yml"]
   - repo: https://github.com/python-jsonschema/check-jsonschema
-    rev: 0.28.1
+    rev: 0.28.4
```

Comment:
Without this `pre-commit` hook update, `pre-commit` will fail as `attestation` is a newer `permissions` key. To keep this atomic I'm only updating this hook instead of all of them.